test/lint.js

```diff
@@ -4,23 +4,6 @@ import atImport from ".."
 
 const processor = postcss().use(atImport())
 
-test("should warn when @import inside block statement", t => {
-  return Promise.all([
-    processor.process(`a { @import ""; }`),
-    processor.process(`@media { @import ""; }`),
-  ])
-  .then(function(results) {
-    results.forEach(function(result) {
-      const warnings = result.warnings()
-      t.is(warnings.length, 1)
-      t.is(
-        warnings[0].text,
-        "@import should not be inside block statement"
-      )
-    })
-  })
-})
-
```

Comment:
I'd like to improve performance here. We don't need to traverse all tree. And imports will be just leaved in block statement.